### PR TITLE
example Spring applications

### PR DESCRIPTION
**Pull request by tarzanek**

### CODE REVIEW

File: spring/springdemo-custom/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Using Scylla driver
+https://java-driver.docs.scylladb.com/stable/
+
+Expects a Scylla DB or Cassandra DB running on
+localhost : 9042
+ev. with cassandra/cassandra auth
+AND with schema loaded from `./schema.cql`
+
+Make sure to adjust src/resources/application.yml , if you will use different Scylla cluster
```
Comment (tzach) [inline spring/springdemo-custom/README.md:20]:
```suggestion
Make sure to adjust src/resources/application.yml , if you will use a different Scylla cluster

```

File: spring/springdemo-custom/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Check `./start-scylla-container.sh` script that can help you run a ScyllaDB single VM
+
+Import to your local Postman ( https://www.postman.com/ ) app a postman collection v 2.1
+`./Springdemo-custom.postman_collection.json`
+to run sample queries against REST API
+
+Request are prepared - hence routed to appropriate node and (cpu) shard thanks to scylla driver.
```
Comment (tzach) [inline spring/springdemo-custom/README.md:28]:
```suggestion
Requests are prepared - hence routed to appropriate node and (cpu) shard thanks to scylla driver.

```

Comment (tzach) [reply spring/springdemo-custom/README.md:28]: or Request is...

File: spring/springdemo-custom/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Prepared statements spring demo on top of Scylla driver
+
+After this is run, you will have a REST API server
```
Comment (tzach) [inline spring/springdemo-custom/README.md:3]: run how?

File: spring/springdemo-custom/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+AsyncStockController lists the respective endpoints
+
+This demo is heavily inspired by
+https://github.com/DataStax-Examples/cassandra-reactive-demo-java/
+which was rewritten to kotlin.
+
+Using Scylla driver
+https://java-driver.docs.scylladb.com/stable/
+
+Expects a Scylla DB or Cassandra DB running on
```
Comment (tzach) [inline spring/springdemo-custom/README.md:15]:
```suggestion
Expects a Scylla or Apache Cassandra running on

```

**Comment by tarzanek** [comment]
thnx @tzach , fixed

**Comment by tzach** [comment]
Worthwhile explicitly stating the version of each sw used (scylla, scylla java driver, spring etc)

**Comment by tarzanek** [comment]
done @tzach

**Comment by tzach** [comment]
LGTM

**Comment by tarzanek** [comment]
@haaawk do you want to review too? or can I merge?

**Review by haaawk — APPROVED** [review]

**Pull request merged**